GirafRest.Test/Services/GirafServiceTest.cs

```diff
@@ -97,7 +108,32 @@ public void CheckProtectedOwnership_DoNotOwnResource_False()
             var res = gs.CheckProtectedOwnership((Pictogram)notUserDepartment.Resources.First().Pictogram, user);
 
             Assert.False(res.Result);
-        }
+        }
         #endregion
+        #region IpRateLimiting
+        [Fact]
+        public async Task CheckIpRateLimiting_ToManyRequests_True_StatusCode429()
+        {
+            _server = new TestServer(new WebHostBuilder().UseStartup<Startup>());
+            _client = _server.CreateClient();
+
+            HttpResponseMessage response = null;
+            for (int i = 0; i < 5; i++)
+            {
+                response = await _client.GetAsync("/");
+            }
+
+            Assert.True(response.StatusCode == (System.Net.HttpStatusCode)429);
+        }
+
+        [Fact]
+        public async Task CheckIpRateLimiting_ToManyRequests_False_StatusCode200()
```

Comment:
The name of this test do not correspond with what is actually tested.

The name states it will return false on having a statusCode 200. But the assertion is that it is false that the statusCode is 429.

Dont we need a test that actually assert that we are able to get a statusCode 200 ?